apps/src/code-studio/components/libraries/LibraryCreationDialog.jsx

```diff
@@ -126,6 +128,17 @@ class LibraryCreationDialog extends React.Component {
     );
   };
 
+  displayUnpublishSuccess = () => {
```

Comment:
Perhaps worth a longer style discussion, but this should maybe be a pure functional React component instead of an inline function

Reply:
✔️